cmd/sync/azure.go

```diff
@@ -181,6 +189,18 @@ func validateAzureConfig(cfg *azureConfig) error {
 		if ups.Kind == "" || !(ups.Kind == "http" || ups.Kind == "stream") {
 			return fmt.Errorf(upstreamKindErrorMsgFormat, ups.Name)
 		}
+		if ups.MaxConns < 0 {
+			return fmt.Errorf(upstreamMaxConnsErrorMsg, ups.MaxConns)
+		}
+		if ups.MaxFails < 0 {
+			return fmt.Errorf(upstreamMaxFailsErrorMsg, ups.MaxFails)
+		}
+		if validateTime(ups.FailTimeout) != nil {
```

Comment:
`validateTime` returns an err, but that err is not used anywhere (that I can think of) so is useless. I suggest one of those approaches:

1. `validateTime` can return a bool true/false instead of err/nil, because the error is not used anywhere
2. Use the error as part f the message in `fmt.Errorf(upstreamFailTimeoutErrorMsg, ups.FailTimeout)`